src/components/Editor/containers/DropdownButtonContainer.js

```diff
@@ -17,7 +17,9 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => {
   return {
-    onSelect: value => dispatch(setMostRecentProgram(value)),
+    onSelect: value => {
```

Comment:
reason for changin the formatting?

Reply:
whoops must have been autoformatted by vscode, can revert